language/vm/vm_runtime/src/loaded_data/function.rs

```diff
@@ -78,7 +78,7 @@ impl<'txn> FunctionReference<'txn> for FunctionRef<'txn> {
     }
 
     fn return_count(&self) -> usize {
-        self.def.local_count
+        self.def.return_count
```

Comment:
@runtian-zhou, @dariorussi: Just giving a shout out to let you know I made a small change to your code and tests to get my bug fix to land.

Reply:
was this a bug? If I understand your fix below you made the test module valid and funny enough nothing in that test broke. Am I correct about it?